### ISSUE TITLE
Shumway seems to be incompatible with NoScript

### ISSUE THREAD

**Issue by castarco** [mozilla/shumway#2287]
When I have Shumway installed on my Firefox, if I open http://www.ant-karlov.ru/games/santa-is-coming.html , shumway loads the .swf file and runs it.

But if in addition I have NoScript installed on the same Firefox instance, then , when I tell NoScript to not block the flash content, Firefox doesn't use Shumway, but the Adobe's Flash plugin.

I don't know if it's a Shumway bug, a NoScript bug, or an unlucky combination of both cases.

**Comment by GKFX** [comment]
Well, Shumway is JavaScript, and NoScript blocks JavaScript, so NoScript seems to be doing its job.

**Comment by castarco** [comment]
Yes, but NoScript allows to unblock some js files... there is no way to unblock shumway scripts?

**Comment by castarco** [comment]
It seems that NoScripts whitelists resource:* and chrome:* . I've uninstalled Adobe's Flash and now Shumway is able to run .swf files even with NoScript installed.

**Comment by LifeIsStrange** [comment]
And what about uBlock Origin ? https://github.com/gorhill/uBlock
